querydsl-sql/src/main/java/com/querydsl/sql/Configuration.java

```diff
@@ -259,7 +259,11 @@ public void setDynamicNameMapping(NameMapping nameMapping) {
                 }
             }
             if (sqlType != null) {
-                stmt.setNull(i, sqlType);
+                if (sqlType == java.sql.Types.OTHER && this.getTemplates().getClass().getSimpleName().contains("Oracle")) {
+                    stmt.setNull(i, Types.STRUCT, "MDSYS.SDO_GEOMETRY");
```

Comment:
This is probably true for Geometry, but what for other types that are represented as `Struct`?